src/Rendering.hs

```diff
@@ -73,41 +110,68 @@ renderPaddle c w h (x, y) = translate x y
                           $ color c
                           $ rectangleSolid w h
 
+-- | render background
+renderBackground :: Color     -- ^ background color
+                 -> Picture   -- ^ picture to render
+renderBackground c = color c
+                    $ rectangleSolid gameWidth gameHeight
+
 -- | render the game
 renderGame :: Game      -- ^ The game state to render
+           -> Library   -- ^ image library
            -> Picture   -- ^ A picture of this game state
 
 -- MainMenu state
-renderGame game @ Game { gameState = MainMenu } = pictures
-      [ renderStateText orange "Haskell" (-120, 100) (0.5, 0.5)
-      , renderStateText orange "Breakout" (-150, 0) (0.5, 0.5)
-      , renderStateText orange "Press ENTER to continue" (-200, -100) (0.25, 0.25)
+renderGame game @ Game { gameState = MainMenu } _ = pictures
+      [ renderBackground stateBGColor
+      , renderStateText stateText "Haskell" (-120, 100) (0.5, 0.5)
```

Comment:
Can't you factor yout some of those `renderStateText stateText` with a `(f)map` ?